lib/elixir/lib/tuple.ex

```diff
@@ -49,6 +49,25 @@ defmodule Tuple do
   end
 
   @doc """
+  Inserts an element into the end of a tuple.
+
+  Returns a new tuple which has one element more than `tuple`, and contains
+  the elements in `tuple` followed by `value` as the last element.
+
+  Inlined by the compiler.
```

Comment:
To actually inline it, you also need to add a clause here:

https://github.com/elixir-lang/elixir/blob/e49a806e8abe43c62c9c1b21ea7eb2217bef1966/lib/elixir/src/elixir_rewrite.erl#L144

Thank you!